import_data/tasks.py

```diff
@@ -10,13 +11,69 @@
 import time
 from datetime import timedelta, datetime
 import requests
-from invoke import task
+from invoke import context, task
 from invoke.exceptions import Failure
 from utils.lock import FileLock
 
+cc_exec = concurrent.futures.ThreadPoolExecutor()
+
 logging.basicConfig(level=logging.INFO)
 
 
+class PrefixedStream:
+    """
+    Wrapper class around a stream adding a prefix to each writen line.
+    """
+    def __init__(self, src_stream, prefix):
+        self.prefix = prefix
+        self.src_stream = src_stream
+        self.buffer = ''
+
+    def write(self, data):
+        self.buffer += data
+        self.flush()
+
+    def flush(self):
+        lines = self.buffer.split('\n')
+
+        for line in lines[:-1]:
+            self.src_stream.write(
+                '\033[0;90m[{}]\033[0;0m {}\n'.format(self.prefix, line)
```

Comment:
These colors will probably not be rendered correctly in our toolchain, so I would stick to raw strings.